apps/ngs_roi/bam2roi.cpp

```diff
@@ -1,7 +1,7 @@
 // ==========================================================================
 //                   NGS: Regions of Interest Analysis
 // ==========================================================================
-// Copyright (c) 2012-2013, Bernd Jagla, Institut Pasteur
```

Comment:
Don't change initial copyright year. And why is the FU not part of the copyright? Is that legally ok? 🤔